core/web/jsonld/PersonInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace luya\web\jsonld;
+
+/**
+ * JsonLd - Person interface
+ *
+ * @see http://schema.org/Organization
+ *
+ * @author Alex Schmid
+ * @since 1.0.0
+ */
+interface PersonInterface
+{
+    /**
+     * Return the fields
+     */
+    public function fields();
```

Comment:
@JohnnyMcWeed fields should not be part of the interface i assume. It should be an abstract method of BaseThing instead.